e2e_tests/tests/experiment/test_model_def.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from tests import config as conf
+from tests import experiment as exp
+
+
+@pytest.mark.e2e_cpu
+def test_fetch_experiment_model_def() -> None:
```

Comment:
question: is there a way to run this as an integration test in go or something?  It seems a bit expensive to _actually_ run an experiment in order to test a REST API for fetching experiment configs?

Reply:
Yes, an integration test has been added at `master/internal/db/cache_intg_test.go`